frontend/dockerfile/docs/reference.md

```diff
@@ -644,22 +644,24 @@ guide](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practi
 
 The cache for `RUN` instructions can be invalidated by [`ADD`](#add) and [`COPY`](#copy) instructions.
 
-## RUN --mount
+### RUN --mount
 
 > **Note**
 >
 > Added in [`docker/dockerfile:1.2`](#syntax)
 
+```dockerfile
+RUN --mount=[type=<TYPE>][,option=<value>[,option=<value>]...]
+```
+
```

Comment:
Note to reviewer: These changes for `RUN` flags are just to keep things consistent between the different commands